app-starters-common/pom.xml

```diff
@@ -24,4 +24,12 @@
 		<module>app-starters-twitter-common</module>
 	</modules>
 
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-java-dsl</artifactId>
+			<version>${spring-integration-java-dsl.version}</version>
+		</dependency>
```

Comment:
OK. Thank you! Good point about `Initializr`. But I really meant exactly this our development process, when we are tied with specific SCSt and Boot version. Right, I might just don't understand something around. Although on the other hand it really might be easier to develop everything in the same version. At least that allow us to avoid jumping around.

Fixed. Fair enough!